profiles/fscloud/README.md

```diff
@@ -1,8 +1,72 @@
+<!-- BEGIN MODULE HOOK -->
 # Financial Services Cloud Profile
 
 This is a profile for MongoDB that meets FS Cloud requirements.
 It has been scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) and meets all applicable goals with the following exception:
 - 3000205: Check whether Databases for MongoDB network access is restricted to a specific IP range.
-    - This is ignored because the CBR locks this down and CRA does not check this
+  - This is ignored because the CBR locks this down and CRA does not check this
 
 ## Note: If no Context Based Restriction(CBR) rules are not passed, you must configure Context Based Restrictions externally to be compliant.
+<!-- END MODULE HOOK -->
+
+<!-- BEGIN EXAMPLES HOOK -->
+## Examples
+- [ Financial Services Cloud profile example](../../examples/fscloud)
+<!-- END EXAMPLES HOOK -->
```

Comment:
Did you add that example manually? Wondering why it only picked up one example. I wasn't expecting examples to be listed here, since they are listed in the main readme already

Reply:
removing "examples" hook